duplicates-in-array.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function (array, n) {
```

Comment:
Does the problem statement mention that you receive two parameters for this function?

Reply:
No it does not say. As I said to Costel on his review, I used n to verify if my given array has the number of items I want and to verify if each item is smaller than n. I could have used only the array's length to verify if each number is smaller than the length, because in the description does not say that I have to check if the array is in correct form.